migrations/20250712_add_monitor_visibility_flags.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Add enable_details_to_be_examined and enable_individual_view_if_grouped
+ * columns to monitors table.
+ */
+export function up(knex) {
+  return knex.schema.alterTable("monitors", function (table) {
+    table.boolean("enable_details_to_be_examined").notNullable().defaultTo(true);
```

Comment:
let us tiny int for this, since there are nuances of how boolean is interpreted for pg/mysql/sqilite